ihmc-graphics/src/libgdx/java/us/ihmc/rdx/tools/RDXModelLoader.java

```diff
@@ -44,6 +46,8 @@ public static Model load(String modelFileName)
 
    /**
     * No synchronization and will load data from file every time.
+    *
+    * FIXME: Cannot load GLTF/GLB to ModelData
```

Comment:
We can't scale the vertex data for GLTFs yet, so models that need to be scaled still need a g3dj.